model_analyzer/triton/server/server_docker.py

```diff
@@ -83,17 +88,29 @@ def start(self):
             server_metrics_port: LOCAL_METRICS_PORT
         }
 
-        # Run the docker container
-        self._tritonserver_container = self._docker_client.containers.run(
-            image=self._tritonserver_image,
-            name='triton-server',
```

Comment:
In L0_server_launch_mode, you will need to get rid of the calls to cleanup.py. There are 2 IIRC.